irispy/spectrograph.py

```diff
@@ -151,3 +156,85 @@ def _enter_column_into_table_as_quantity(header_property_name, header, header_co
         raise ValueError("Multiple property names equal to {0}".format(header_property_name))
     pop_colname = header_colnames.pop(index)
     return Quantity(data[:, header[pop_colname]], unit=unit)
+
+
+class SpectrogramSequence(NDCubeSequence):
+    """docstring for SpectrogramSequence"""
+
+    def __init__(self, data_list, meta=None, common_axis=None, **kwargs):
```

Comment:
For this object, ```common_axis``` should be a required arg, not a kwarg.  Also, I think ```raster_positions_per_scan``` and ```first_exposure_raster_position``` should be user args, as the shape of the underlying ```NDCubeSequence``` may not reflect the rastering mode itself.